docs/scripting/resources/vehiclecolorid.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Vehicle Color IDs
+
+description:
```

Comment:
Empty description, either write one or remove the tag.

Reply:
Done removed the tag.